tasks/utils.py

```diff
@@ -151,57 +151,47 @@ def get_git_branch_name():
     return check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).strip()
 
 
-def query_version(ctx, git_sha_length=7, match=None):
+def query_version(ctx, git_sha_length=7, dca=False):
     # The string that's passed in will look something like this: 6.0.0-beta.0-1-g4f19118
     # if the tag is 6.0.0-beta.0, it has been one commit since the tag and that commit hash is g4f19118
     cmd = "git describe --tags"
-    if match:
-        cmd += " --match \"{}\"".format(match)
+    if dca:
+        cmd += " --match \"{}\"".format('dca-*')
+    if git_sha_length and type(git_sha_length) == int:
+        cmd += " --abbrev={}".format(git_sha_length)
```

Comment:
The `describe` command will only look up the 10 most recent tags. I think we could increase this value as we build more and more RC and bugfix (with the `--candidates=` option). I doubt that the DCA could build their latest version after a release week.

Reply:
I didn't know that, thanks!